src/main/java/org/mskcc/oncokb/model/TrialJson.java

```diff
@@ -0,0 +1,78 @@
+package org.mskcc.oncokb.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+
+/**
+ * TrialJson
+ */
+@javax.annotation.Generated(value = "io.swagger.codegen.languages.SpringCodegen", date = "2018-02-23T17:47:44.538-05:00")
+
+public class TrialJson {
```

Comment:
Because curation platform can provide a trial json object in good format, we can use it to build json file directly. I think it's better than building a Trial model since a trial is so dynamic in `match` clause.

Reply:
Then why do we even create this model?

Reply:
This model is created by swagger and used to store trial json sent from firebase.